src/frontend/templates/pages/networks.html

```diff
@@ -1,57 +1,61 @@
-  officia deserunt mollitia animi, id est laborum et dolorum fuga. Et harum quidem rerum facilis est et expedita
-  distinctio. Nam libero tempore, cum soluta nobis est eligendi optio cumque nihil impedit quo minus id quod maxime
-  placeat facere possimus, omnis voluptas assumenda est, omnis dolor repellendus. Temporibus autem quibusdam et aut
-  officiis debitis aut rerum necessitatibus saepe eveniet ut et voluptates repudiandae sint et molestiae non recusandae.
-  Itaque earum rerum hic tenetur a sapiente delectus, ut aut reiciendis voluptatibus maiores alias consequatur aut
-  perferendis doloribus asperiores repellat."
-  <h1 class="title">1914 translation by H. Rackham</h1>
-
-  "On the other hand, we denounce with righteous indignation and dislike men who are so beguiled and demoralized by the
-  charms of pleasure of the moment, so blinded by desire, that they cannot foresee the pain and trouble that are bound
-  to ensue; and equal blame belongs to those who fail in their duty through weakness of will, which is the same as
-  saying through shrinking from toil and pain. These cases are perfectly simple and easy to distinguish. In a free hour,
-  when our power of choice is untrammelled and when nothing prevents our being able to do what we like best, every
-  pleasure is to be welcomed and every pain avoided. But in certain circumstances and owing to the claims of duty or the
-  obligations of business it will frequently occur that pleasures have to be repudiated and annoyances accepted. The
-  wise man therefore always holds in these matters to this principle of selection: he rejects pleasures to secure other
-  greater pleasures, or else he endures pains to avoid worse pains."
+{% if not run %}
+No currently active run to view networks for.
+{% elif viewing_current_run %}
+<h1 class="title">Networks for Current Run ({{run.name}}) </h1>
+{% else %}
+<h1 class="title">Networks for {{run.name}}</h1>
+{% endif %}
+
+{% if run and object_list|length <= 0 %}
+No networks found for this run!
+{% elif run and object_list|length > 0 %}
+<table>
+  <tr>
+    <th> Name </th>
+    <th> Date </th>
+    <th> Approx Elo Rating </th>
+    <th> Download </th>
+    <th> Extra Notes </th>
+  </tr>
+  {% for network in object_list %}
+  <tr>
+    <td> {{ network.name }} </td>
+    <td> {{ network.created_at|date:"Y-m-d H-i-s e" }} </td>
+    <td> {{ network.rating }} </td>
+
+    {% if network.model_zip_file %}
+    <td> <a href="{{ network.model_zip_file.url }}">[zip]</a> </td>
+    {% elif network.model_file %}
+    <td> <a href="{{ network.model_file.url }}">[bin.gz]</a> </td>
+    {% else %}
+    <td> - </td>
+    {% endif %}
+
+    <td> {{ network.notes }} </td>
+  </tr>
+  {% endfor %}
+</table>
+{% endif %}
+
+{% if all_runs_to_show|length > 0 %}
+<h1 class="title"> Networks By Run </h1>
+<ul>
+  {% for run in all_runs_to_show %}
+  <li><a href="{% url 'networks' run.name %}"> {{ run.name }}</a></li>
+  {% endfor %}
+</ul>
+{% endif %}
```

Comment:
This is used when:

* There is more than one run. In that case, this provides a table of links to the networks for older runs besides the current run.

OR

* There is no current run because none of the runs are active right now (even if there is only 1 run). In that case, this provides a table of links to the networks for all runs.